Src/Fido2.Models/Fido2Configuration.cs

```diff
@@ -56,20 +56,15 @@ public Fido2Configuration()
     /// <summary>
     /// Server origins, including protocol host and port.
     /// </summary>
-    public ISet<string> Origins
+    public IReadOnlySet<string> Origins
     {
         get
         {
             if (_origins == null)
             {
-                _origins = new HashSet<string>();
-
-                // Since we're depricating Origin we ease the transition to move the value automatically, unless its null
+                // Since we're deprecating Origin we ease the transition to move the value automatically, unless its null
 #pragma warning disable CS0618 // Type or member is obsolete
-                if (Origin != null)
-                {
-                    _origins.Add(Origin);
-                }
+                _origins = new HashSet<string>(new[]{Origin});
```

Comment:
This would add null to `Origins` if `Origin` is null, is that really what we want? I would prefer it to be empty.

Reply:
Oh yes definitely, I updated it.